docs/tikz/arch-multi-standby.tex

```diff
@@ -1,78 +1,103 @@
+	  align=center,
+	  rectangle split part align={center, center, left},
+	  minimum height=2.5cm,minimum width=4cm,inner sep=0.5cm]
 
-  \tikzstyle{standby}=[rectangle split,rectangle split parts=3,
-    text=white,draw=white,fill=blue!25,text=black!90,
-    minimum height=1.75cm,minimum width=4cm]
+  \tikzstyle{primary}=[mpnode,text=ptxt,draw=white,
+  	  rectangle split part fill={pbox,pbox,white}]
 
-  \tikzstyle{monitor}=[draw=green!35,fill=green!35,text=black!90,
-    minimum height=1.5cm,minimum width=3cm]
+  \tikzstyle{standby}=[mpnode,text=stxt,draw=white,
+	  rectangle split part fill={sbox,sbox,white}]
+
+  \tikzstyle{monitor}=[node,text=mtxt,draw=mbox,fill=mbox]
 
   %% \draw [help lines] (-10,0) grid (10,20);
 
-  \node  (a)   at (0,17)   [primary]
-         {Node A
+  \node (flegend) at (0,18) {\textt{number\_sync\_standby = 1}} ;
+
+  \node  (a)   at (0,15)   [primary]
+         {\textbf{\normalsize Node A}
            \nodepart{second}
-           \textbf{Primary}
-           \nodepart{third}
-           \texttt{number\_sync\_standby = 1}
-           \nodepart[align=left]{fourth}
-           \texttt{replication quorum = true} \\
-           \texttt{candidate priority = 50}
+           \textbf{\Large Primary}
+           \nodepart[text=stxt]{third}
+		   \texttt{replication quorum = true} \\
+		   \texttt{candidate priority = 50}
          };
 
-  \node  (b)   at (-6,12)  [standby,text width=5.5cm,align=center]
-         {Node B
+  \node  (b)   at (-6,8)  [standby]
+         {\textbf{\normalsize Node B}
            \nodepart{second}
-           \textbf{Secondary}
-           \nodepart[align=left]{third}
-           \texttt{replication quorum = true} \\
-           \texttt{candidate priority = 50}
+           \textbf{\Large Secondary}
+           \nodepart{third}
+		   \texttt{replication quorum = true} \\
+		   \texttt{candidate priority = 50}
          };
 
-  \node  (c)   at (6,12)   [standby,text width=5.5cm,align=center]
-         {Node C
+  \node  (c)   at (6,8)   [standby]
+         {\textbf{\normalsize Node C}
            \nodepart{second}
-           \textbf{Secondary}
-           \nodepart[align=left]{third}
-           \texttt{replication quorum = true} \\
-           \texttt{candidate priority = 50}
+           \textbf{\Large Secondary}
+           \nodepart{third}
+		   \texttt{replication quorum = true} \\
+		   \texttt{candidate priority = 50}
          };
 
-  \node  (app) at (-6,18)  [app]        {Application};
-  \node  (m)   at (6,18)   [monitor]    {Monitor};
+  \node  (app) at (-8,18)  [app]        {\textbf{\Large Application}};
+  \node  (m)   at (8,18)   [monitor]    {\textbf{\Large Monitor}};
 
   \tikzstyle{sql}=[->,color=blue!60,text=black!90,line width=0.15em]
   \tikzstyle{sqlf}=[->,color=blue!35,text=black!90,line width=0.15em,loosely dashed]
   \tikzstyle{sr}=[>->,color=black!70,text=black!90,line width=0.15em]
-  \tikzstyle{hc}=[<->,color=green!50,text=black!90,line width=0.15em,dotted]
```

Comment:
The arrows still use the old colors. I guess these should also be changed, right?

Reply:
Also good point, fixed to use the same colour as the boxes now.